Userland/Libraries/LibJS/Runtime/StringPrototype.h

```diff
@@ -17,7 +17,7 @@ struct CodePoint {
 };
 
 CodePoint code_point_at(Utf16View const& string, size_t position);
-static constexpr Utf8View whitespace_characters = Utf8View("\x09\x0A\x0B\x0C\x0D\x20\xC2\xA0\xE1\x9A\x80\xE2\x80\x80\xE2\x80\x81\xE2\x80\x82\xE2\x80\x83\xE2\x80\x84\xE2\x80\x85\xE2\x80\x86\xE2\x80\x87\xE2\x80\x88\xE2\x80\x89\xE2\x80\x8A\xE2\x80\xAF\xE2\x81\x9F\xE3\x80\x80\xE2\x80\xA8\xE2\x80\xA9\xEF\xBB\xBF"sv);
```

Comment:
Isn't `explicit constexpr Utf8View(StringView string)` a valid constructor for this?

In fact I think that static constexpr forces it to be constexpr but that I'm not sure about.

Reply:
You're right! Turns out, I misread the compilation error message, and it's missing an include instead.

Weird, I could have sworn that the compilation error message said something else.